modules/imgproc/src/color.simd_helpers.hpp

```diff
@@ -104,6 +104,10 @@ struct CvtHelper
             CV_Assert( sz.width % 2 == 0 && sz.height % 3 == 0);
             dstSz = Size(sz.width, sz.height * 2 / 3);
             break;
+        case FROM_UYVY:
+            CV_Assert( sz.width % 2 == 0);
```

Comment:
Shouldn't it be `(scn * sz.width) % 4 == 0`? (currently `scn` has to be 2 but, as I understand, it can in theory be 1 or 4 too)